ajax/libs/react/package.json

```diff
@@ -13,19 +13,18 @@
     "transformer",
     "view"
   ],
-  "homepage": "http://facebook.github.io/react",
-  "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/reactjs/react-bower.git",
-    "fileMap": [
-      {
-        "basePath": "",
-        "files": [
-          "*.js"
-        ]
-      }
-    ]
-  },
+  "homepage": "https://facebook.github.io/react/",
+  "npmName": "react",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "cjs/*,
+        "umd/*"
```

Comment:
Yes, but initial builds CI failed saying it couldn't find it. So was trying other approaches..

Directory structure is as follows:

```
.
├── cjs
│   ├── react.development.js
│   └── react.production.min.js
├── index.js
├── LICENSE
├── package.json
├── PATENTS
├── README.md
└── umd
    ├── react.development.js
    └── react.production.min.js
```